### PR TITLE
[devtools-translate] Japanese - What's New in DevTools (Chrome 93)

### LINKED ISSUE #1295: [devtools-translate] Japanese - What's New in DevTools (Chrome 93)
Post: https://developer.chrome.com/blog/new-in-devtools-93/

Translator: @technohippy
Reviewer: @yoichiro, @lacolaco

### PR DESCRIPTION
**Pull request by technohippy**
Fixes #1295

Changes proposed in this pull request:

- Japanese version of New in devtools 93

### CODE REVIEW

File: site/ja/blog/new-in-devtools-93/index.md [hunk 1]
```diff
@@ -0,0 +1,250 @@
+---
+layout: "layouts/blog-post.njk"
+title: "What's New In DevTools (Chrome 93)"
```
Comment (chybie) [inline site/ja/blog/new-in-devtools-93/index.md:3]: Please help to translate the line into Japanese.

File: site/ja/blog/new-in-devtools-93/index.md [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+authors:
+  - jecelynyeen
+date: 2021-07-28
+updated: 2021-07-28
+description:
+  "Editable CSS container queries, web bundle preview, better string handling in the Console and more."
```
Comment (chybie) [inline site/ja/blog/new-in-devtools-93/index.md:9]: Please help to translate the line into Japanese.

File: site/_includes/partials/devtools/ja/whats-new.md [hunk 3]
```diff
@@ -1,3 +1,15 @@
 ## More DevTools features {: #whats-new }
 Please refer to the English version of [What's New In DevTools](/tags/new-in-devtools/) for a complete list of released features. Below are some content that has been translated into Japanese.
```
Comment (chybie) [inline site/_includes/partials/devtools/ja/whats-new.md:2]: Please help to translate the line into Japanese.

File: site/ja/blog/new-in-devtools-93/index.md [hunk 4]
```diff
@@ -0,0 +1,248 @@
...
+-->
+**Network**パネルでJSONレスポンスを開き、`{}`アイコンをクリックするとプリティプリントされます。
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/x2NKXwJPzjycjeD7cLH6.png", alt=" Pretty-printing JSON responses in the Network panel", width="800", height="523" %}
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/250c88b4d02da283cd0a96204b1592f59fda2fcb #}
+
+Chromium bug: [998674](https://crbug.com/998674) 
+
+{% include 'partials/devtools/ja/reach-out.md' %}
```
Comment (chybie) [inline site/ja/blog/new-in-devtools-93/index.md:247]: Please help to translate this file content into Japanese.

**Review by chybie — CHANGES_REQUESTED** [review]

**Comment by chybie** [comment]
@yoichiro, @lacolaco, please help to take a look!

File: site/ja/blog/new-in-devtools-93/index.md [hunk 5]
```diff
@@ -0,0 +1,250 @@
...
+
+
+<!--
+## Pretty-printing JSON responses in the Network panel {: #pretty-print-json }
+-->
+## NetworkパネルでのJSONレスポンスのプリティプリント {: #pretty-print-json }
+<!--
+You can now pretty print JSON responses in the **Network** panel.
+-->
+NetworkパネルでJSONレスポンスがプリティプリントされようになりました。
```
Comment (lacolaco) [inline site/ja/blog/new-in-devtools-93/index.md:235]:
```suggestion
NetworkパネルでJSONレスポンスがプリティプリントされるようになりました。
```

**Review by lacolaco — COMMENTED** [review]
全体的に読みやすかったです

File: site/ja/blog/new-in-devtools-93/index.md [hunk 6]
```diff
@@ -0,0 +1,250 @@
...
+
+<!--
+If your site exposes source maps to Lighthouse, look for the **View Treemap** button to see a breakdown of your shipped JavaScript, filterable by size and coverage on load. 
+-->
+対象のサイトがLighthouseにソースマップを公開している場合、**View Treemap**ボタンを探すと、配信されたJavaScriptのブレークダウンを、サイズとロード時のカバレッジでフィルタして確認できます。
+
+<!--
+The report also includes a new metric filter (Refer to the **Show audits relevant to** filter in the screenshot). Pick a metric to focus on the opportunities and diagnostics most relevant to improving just that metric.
+-->
+レポートには新しいメトリクスフィルタも追加されています（スクリーンショットの**Show audits relevant to**フィルタを参照）。そのメトリクスだけを改善するためにもっとも適切なOpportunitiesとDiagnosticsにフォーカスを当てるためにメトリクスを選択してください。
```
Comment (yoichiro) [inline site/ja/blog/new-in-devtools-93/index.md:161]:
```suggestion
レポートには新しいメトリクスフィルタも追加されています（スクリーンショットの**Show audits relevant to**フィルタを参照）。そのメトリクスだけを改善するための最も適切なOpportunitiesとDiagnosticsにフォーカスを当てるためにメトリクスを選択してください。
```

**Review by yoichiro — CHANGES_REQUESTED** [review]
@technohippy いくつかコメントしました。ご確認ください。

File: site/ja/blog/new-in-devtools-93/index.md (hunk 4 above)
Comment (technohippy) [reply site/ja/blog/new-in-devtools-93/index.md:249]: done

File: site/ja/blog/new-in-devtools-93/index.md [hunk 7]
```diff
@@ -0,0 +1,250 @@
...
+## StylesパネルでCSSコンテナクエリを編集 {: #container-queries }
+<!--
+You can now view and edit [CSS container queries](https://web.dev/new-responsive/#responsive-to-the-container) in the **Styles** pane.
+-->
+**Styles**パネルで[CSSコンテナクエリ](https://web.dev/new-responsive/#responsive-to-the-container)を確認して編集できるようになりました。
+
+<!--
+Container queries provide a much more dynamic approach to responsive design. The `@container` at-rule works in a similar way to a media query with `@media`. However, instead of querying the viewport and user agent for information, `@container` queries the ancestor container that matches certain criteria.
+-->
+コンテナクエリを使用するとレスポンシブデザインをよりダイナミックに実現できます。@-規則の`@container`はメディアクエリの`@media`と同じように動作しますが、`@media`のようにビューポートとユーザーエージェントに問い合わせて情報を得るのではなく、`@container`は祖先コンテナ要素にある基準に合致するかどうかを問い合わせます。
```
[the remark this answers is not in the document]
Comment (technohippy) [reply site/ja/blog/new-in-devtools-93/index.md]: done

[the remark this answers is not in the document]
Comment (technohippy) [reply site/ja/blog/new-in-devtools-93/index.md]: done

**Comment by technohippy** [comment]
@googlebot I consent.

**Comment by yoichiro** [comment]
@googlebot I consent.

**Comment by lacolaco** [comment]
@googlebot I consent.

**Comment by technohippy** [comment]
@chybie Please manually confirm consent of "ANDO Yasushi" and set the cla label to yes because it's me.

**Comment by chybie** [comment]
In Chinese, when we write English word in Chinese sentence, we will add spaces **before and after** the English word. For example, *我们的 DevTools 加了新功能。*

Is there such a practise in Japanese? @technohippy @lacolaco @yoichiro

**Comment by myakura** [comment]
Hi @chybie ,

> In Chinese, when we write English word in Chinese sentence, we will add spaces **before and after** the English word. For example, _我们的 DevTools 加了新功能。_ (Notice the spacing before and after "DevTools")

Some people (especially tech) do that in Japanese, too. Japanese MDN contributors has [a guideline to include spaces around latin alphabet words](https://github.com/mozilla-japan/translation/wiki/L10N-Guideline#%E8%8B%B1%E6%95%B0%E8%A8%98%E5%8F%B7%E3%81%A8%E5%85%A8%E8%A7%92%E5%8D%8A%E8%A7%92). Whether to include spaces or not, it will be so nice to have some guidelines on style!

File: site/ja/blog/new-in-devtools-93/index.md [hunk 8]
```diff
@@ -0,0 +1,250 @@
...
+  "編集可能なCSSコンテナクエリ、Web bundleプレビュー、コンソールでの文字列の扱いの改善など。"
+hero: 'image/dPDCek3EhZgLQPGtEG3y0fTn4v82/xFq1Fb2KOrQfq1RG6x5e.jpg'
+alt: ''
+tags:
+  - new-in-devtools
+  - devtools
+  - chrome-93
+---
+
+*Thanks [technohippy](https://github.com/technohippy) for the translations, [yoichiro](https://github.com/yoichiro) and [lacolaco](https://github.com/lacolaco) for the reviews!*
```
[the remark this answers is not in the document]
Comment (yoichiro) [reply site/ja/blog/new-in-devtools-93/index.md:18]: @chybie The `yoichiro` looks good to me without any modifications.

[the remark this answers is not in the document]
Comment (lacolaco) [reply site/ja/blog/new-in-devtools-93/index.md:18]: No problem, too.

File: site/ja/blog/new-in-devtools-93/index.md [hunk 9]
```diff
@@ -0,0 +1,250 @@
...
+## Attribution Reporting APIデバッグ {: #attribution-reporting }
+<!--
+Attribution Reporting API errors are now reported in the **Issues** tab.
+-->
+Attribution Reporting APIのエラーが**Issues**タブでレポートされるようになりました。
+
+<!--
+[Attribution Reporting](https://developer.chrome.com/docs/privacy-sandbox/attribution-reporting/) is a new API to help you measure when a user action (such as an ad click or view) leads to a conversion, without using cross-site identifiers.
+-->
+[Attribution Reporting](https://developer.chrome.com/docs/privacy-sandbox/attribution-reporting/)は（広告のクリックや視聴のような）ユーザーアクションがコンバージョンを引き起こすタイミングについて、クロスサイトの識別を伴わずに測定するための新しいAPIです。
```
Comment (agektmr) [inline site/ja/blog/new-in-devtools-93/index.md:90]:
```suggestion
[Attribution Reporting](https://developer.chrome.com/docs/privacy-sandbox/attribution-reporting/)は（広告のクリックや視聴のような）ユーザーアクションがコンバージョンを引き起こすタイミングについて、クロスサイトの識別子を伴わずに測定するための新しいAPIです。
```

File: site/ja/blog/new-in-devtools-93/index.md [hunk 10]
```diff
@@ -0,0 +1,250 @@
...
+## CORSデバッグの改善 {: #cors }
+<!--
+CORS-related TypeErrors in the **Console** are now linked to the Network panel and Issues tab. 
+-->
+**Console**のCORSに関連するTypeErrorがNetworkパネルとIssueタブにリンクされるようになりました。
+
+<!--
+Click on the two new icons next to the CORS-related error message to view the network request, or understand the error message further and get potential solutions in the Issues tab.
+-->
+CORSに関連するエラーメッセージの横にある2つの新しいアイコンをクリックすると、ネットワークリクエストを確認したり、Issueタブでエラーメッセージの詳細と想定される解決策を確認できます。
```
Comment (agektmr) [inline site/ja/blog/new-in-devtools-93/index.md:138]:
```suggestion
CORSに関連するエラーメッセージの横にある2つの新しいアイコンをクリックすると、ネットワークリクエストを確認したり、Issueタブでエラーメッセージの詳細と、想定される解決策を確認できます。
```

**Review by agektmr — APPROVED** [review]
LGTM % nit

**Comment by technohippy** [comment]
@chybie I'm not sure what's a problem but the translation has been done anyway. Please move things forward.

**Review by chybie — APPROVED** [review]

**Comment by chybie** [comment]
Thanks @technohippy and all! Merged. Will be available on this url soon: https://developer.chrome.com/ja/blog/new-in-devtools-93/. :)

**Pull request merged**